### DISCUSSION TITLE
CKAN Preview

### DISCUSSION THREAD

**Discussion by Joebjammin** [TerriaJS/terriajs#5965] [General] [answered]
Hi Folks,
Are there any plans to implement a terriajs 8 CKAN preview?
Regards,
...Joe

**Comment by nf-s** [comment, marked as answer]
Hi @Joebjammin 

I'm not currently aware of any plans to upgrade the CKAN preview for terriajs version 8. @AnaBelgun will be able to tell you if there are.

The latest version of terriajs will automatically detect older v7 configuration and upgrade it to v8 using our [catalog-converter](https://github.com/TerriaJS/catalog-converter). It is far from perfect, but this means it should handle most datasets displayed through the CKAN preview plugin reasonably well.

If you have examples of public CKAN datasets that you can provide, I can give them a test to show you if the automatic conversion is suitable for you.

Thanks,
Nick
